.env.sample

```
@@ -3,4 +3,4 @@
 # MONGO_URL=mongodb://localhost:27017/erxes
 # NODE_OPTIONS=--max-old-space-size=1536
 # ENABLED_PLUGINS=operation
-# DISABLE_CHANGE_STREAM=true
+# DISABLE_CHANGE_STREAM=true
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)